file_snapshot.go

```diff
@@ -384,6 +384,11 @@ func (s *FileSnapshotSink) Close() error {
 	// Close the open handles
 	if err := s.finalize(); err != nil {
 		s.logger.Printf("[ERR] snapshot: Failed to finalize snapshot: %v", err)
+		s.logger.Printf("[INFO] snapshot: Deleting temporary files...")
+		if delErr := os.RemoveAll(s.dir); delErr != nil {
```

Comment:
This is definitely a temp dir, right? I looked at the code and I think this is fine - just want to double check it doesn't blow away snapshots.